plots/measure.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const path = require('path');
+const parseURL = require('url').parse;
+
+const mkdirp = require('mkdirp');
+
+const constants = require('./constants.js');
+const utils = require('./utils.js');
+const config = require('../lighthouse-core/config/plots.json');
+const lighthouse = require('../lighthouse-core/index.js');
+const ChromeLauncher = require('../lighthouse-cli/chrome-launcher.js').ChromeLauncher;
+const Printer = require('../lighthouse-cli/printer');
+const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
+
+const NUMBER_OF_RUNS = 20;
+
+const URLS = [
```

Comment:
> It's about ~20 sec per run * 72 sites * 20 runs = 8 hours. We can create a narrower list (e.g. the most problematic sites) when we generate plots as part of the lighthouse's CI process

yeah, I like that. Maybe we can have it be a few sets of URLs, the full run does all of them, and then flags for different subsets (and even a single URL)